css-fonts/styles.css

```diff
@@ -0,0 +1,10 @@
+h1 {
+  font-family: 'Bebas Neue', sans-serif;
+  line-height: 150%;
+}
+
+h2,
```

Comment:
Your `h1` & `h2` should be using the same `font-family`

Reply:
<img width="1186" alt="Screenshot 2023-07-19 at 4 41 13 PM" src="https://github.com/julieluangpraxay/c0723-code-solutions/assets/130415142/330ca592-e7c7-4f24-9987-2737ef2be742">